tests/e2e/vLLM/test_lmeval.py

```diff
@@ -0,0 +1,131 @@
+import os
+import shutil
+from pathlib import Path
+
+import numpy
+import pytest
+import yaml
+from loguru import logger
+
+from llmcompressor.core import active_session
+from tests.e2e.e2e_utils import run_oneshot_for_e2e_testing
+from tests.examples.utils import requires_gpu_count
+
+try:
+    import lm_eval
+
+    lm_eval_installed = True
+except ImportError:
+    lm_eval_installed = False
+    logger.warning("lm_eval is not installed. This test will be skipped")
+
+TEST_DATA_FILE = os.environ.get("TEST_DATA_FILE", None)
+
+
+# Will run each test case in its own process through run_tests.sh
+# emulating vLLM CI testing
+@requires_gpu_count(1)
+@pytest.mark.skipif(
+    not lm_eval_installed, reason="lm eval is not installed, skipping test"
+)
+class TestLMEval:
```

Comment:
Do you not need to inherit unittest here?

Reply:
No, using pyest

Reply:
We don't need to since pytest can pick up both!